tsconfig.node.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
not sure we need it, we can add it to the main one

Reply:
deleted